### PR TITLE
89 better next steps

### PR DESCRIPTION
**Pull request by snoyberg**
The "How to Build" tutorial isn't yet written, but I've copied @soupi's tutorial for "How to Play" and wrote up a quick "How to Script".

### CODE REVIEW

**Comment by chrisdone** [comment]
I like the How to Script with Stack. Nice.

I might put some `[snip]`'s in the big outputs in Stack Play, though.

**Comment by snoyberg** [comment]
I fleshed out a Stack Build tutorial, and added some `[snip]`s to the Stack Play doc. @chrisdone look good for merge?

File: static/tutorial/stack-build.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    executable destination, and your build tools (like `ghc`).
+
+2.  `stack exec which my-executable` will use the `which` command to
+    find the full path to your executable, which you could then run,
+    without the additional modifications that `stack exec` implies. If
+    you want to be clever, you could do something like this from your
+    shell:
+
+    ```shell
+    $ $(stack exec which my-executable)
```
Comment (chrisdone) [inline static/tutorial/stack-build.md:93]: Syntax error here, shows up in the display: https://github.com/haskell-lang/haskell-lang/blob/2561f72671b596bf44dda751755bfeb79931ae06/static/tutorial/stack-build.md#running-executables

Comment (chrisdone) [reply static/tutorial/stack-build.md:93]: Oh nice, I can edit in the page directly. That makes reviewing easier.

**Comment by chrisdone** [comment]
Looks good now!

**Pull request merged**